.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
The `ISSUE_TEMPLATE` AND `PULL_REQUEST_TEMPLATE` can be removed from the package as they are automatically inherited from the organisation .github repository

Reply:
This will be addressed in https://github.com/epiverse-trace/vaccineff/issues/88